test/user.e2e-spec.ts

```diff
@@ -365,4 +366,135 @@ describe('User e2e', () => {
+      displayFirstName: faker.name.firstName(),
+      displayLastName: faker.name.lastName(),
+      password: faker.internet.password(),
+      phone: faker.phone.phoneNumber(),
+      timezone: 'timezone detail',
+      bio: 'bio detail',
+      status: UserStatus.Active,
+    };
+    const newUser = await createUser(app, fakeUser);
+    const org = await createOrganization(app);
+
+    const result = await app.graphql.mutate(
+      gql`
+        mutation assignOrganizationToUser(
+          $orgId: ID!
+          $userId: ID!
+          $primary: Boolean!
+        ) {
+          assignOrganizationToUser(
+            input: {
+              request: { orgId: $orgId, userId: $userId, primary: $primary }
+            }
+          )
+        }
+      `,
+      {
+        orgId: org.id,
+        userId: newUser.id,
+        primary: true,
+      }
+    );
+
+    expect(result.assignOrganizationToUser).toBe(true);
+
+    const result1 = await app.graphql.query(
+      gql`
+        query user($id: ID!) {
+          user(id: $id) {
+            ...user
+            organizations {
+              items {
+                ...org
+              }
+              hasMore
+              total
+              canRead
+              canCreate
+            }
+          }
+        }
+        ${fragments.user}
+        ${fragments.org}
+      `,
+      {
+        id: newUser.id,
+      }
+    );
+    const actual: User = result1.user;
+    expect(actual).toBeTruthy();
+    return true;
+  });
+
+  it('read one users education', async () => {
+    const newUser = await createUser(app);
+    await createEducation(app, { userId: newUser.id });
+
+    const result = await app.graphql.query(
+      gql`
+        query user($id: ID!) {
+          user(id: $id) {
+            ...user
+            education {
+              items {
+                ...education
+              }
+              hasMore
+              total
+              canRead
+              canCreate
+            }
+          }
+        }
+        ${fragments.user}
+        ${fragments.education}
+      `,
+      {
+        id: newUser.id,
+      }
+    );
+    const actual = result.user;
+    expect(actual).toBeTruthy();
+    return true;
+  });
+
+  it('read one users unavailablity', async () => {
+    const newUser = await createUser(app);
+    await createUnavailability(app, { userId: newUser.id });
+
+    const result = await app.graphql.query(
+      gql`
+        query user($id: ID!) {
+          user(id: $id) {
+            ...user
+            unavailabilities {
+              items {
+                ...unavailability
+              }
+              hasMore
+              total
+              canRead
+              canCreate
+            }
+          }
+        }
+        ${fragments.user}
+        ${fragments.unavailability}
+      `,
+      {
+        id: newUser.id,
+      }
+    );
+    const actual = result.user;
+    expect(actual).toBeTruthy();
```

Comment:
I need an expect statement here to test that the unavailability on the user is returned here.